ocs_ci/deployment/aws.py

```diff
@@ -576,7 +591,7 @@ def run_ansible_playbook(self):
             rhel_pod_obj.exec_cmd_on_node(
                 host,
                 pem_dst_path,
-                f'sudo mv {os.path.join("/tmp", repo_file)} {repo_dst_path}',
+                f'sudo mv {os.path.join("/tmp", repo_file)} {constants.YUM_REPOS_PATH}',
```

Comment:
The `/tmp` here and few lines above could be probably replaced by constants `POD_UPLOADPATH` or `RHEL_TMP_PATH`, but I think it is not necessary to change it in this PR.